README.md

```diff
@@ -4,4 +4,101 @@ This repository provides the data and scripts associated with the [Zika virus tu
 
 See the [original Zika build repository](https://github.com/nextstrain/zika) for more about the public build.
 
-This repo is a conversation of the standard Snakemake workflow into Nextflow.
+This repo is a conversion of the standard Snakemake workflow into [Nextflow](https://www.nextflow.io/).
+
+As a potential joint effort, the [nf-core](https://nf-co.re/) community attempts to collect and curate nextflow analysis pipelines. Their slack channel is very active and provides insight on design decisions of the dsl. Their github provides a workflow template and github action linters.
+
+## Demonstration
+
+Augur commands were wrapped in processes (similar to Snakemake's rules) and placed in the `modules/augur.nf`. Nextflow processes were imported into `main.nf` and connected via Nextflow channels.
+
+```
+sequence_ch | index |                                       // INDEX
+  combine(metadata_ch) | combine(exclude_ch) | filter |     // FILTER
+  combine(reference_ch ) | align |                         // ALIGN
+  tree |                                                   // TREE
+  combine(align.out) | combine(metadata_ch) | refine        // REFINE
+...
+```
+
+See `main.nf` for full details.
+
+To run the workflow:
+
+```
+# (1) Install nextstrain or activate the nextstrain conda environment
+conda activate nextstrain
```

Comment:
[non-blocking] Note that this only works on native runtime, since [the current image](https://github.com/nextstrain/docker-base/blob/b3ba871576a40f1e30fbb0a1211b3a44e5de56f1/Dockerfile) for Docker runtime does not have Nextflow installed.

(We'll soon have a glossary page explaining what a runtime is, which can then be referenced here.)

Reply:
Good to know! Thank you! 🙏 😄